### PR TITLE
Sscs 3652 content for DWP respond for paper case

### PR DESCRIPTION
**Pull request by barbaracassani**
Includes:
- SSCS-3599
- Sscs 3652
- SSCS-3464

### CODE REVIEW

File: test/unit/core/contentLookup.test.js [hunk 1]
```diff
@@ -48,13 +48,22 @@ describe('Calling the getContentAsString() function', () => {
 describe('Calling the getContentAsArray() function', () => {
   it('should retrieve the content of all events as an array', () => {
     allEventKeys.forEach(key => {
-      const contentArr = getContentAsArray(`status.${key}.content`);
+      const contentArr = getContentAsArray(`status.${key}.oral.content`);
       return expect(contentArr).to.be.an('array').that.is.not.empty;
     });
   });
   it('should...', () => {
```
Comment (guioconnor) [inline test/unit/core/contentLookup.test.js]: These two tests should have clearer descriptions.

Comment (barbaracassani) [reply test/unit/core/contentLookup.test.js]: True! Shall amend 👍

Comment (barbaracassani) [reply test/unit/core/contentLookup.test.js]: 👍

**Review by guioconnor — APPROVED** [review]

**Review by Edlyne — APPROVED** [review]
QA APPROVED.

**Review by Edlyne — APPROVED** [review]
QA APPROVED.

**Pull request merged**